foundations/javascript_basics/javascript_developer_tools.md

```diff
@@ -45,7 +45,6 @@ Google has updated some of the required sections in the below tutorials and some
     -  JavaScript
         1.  [Debug JavaScript](https://developer.chrome.com/docs/devtools/javascript/)
         2.  [Pause your code with breakpoints](https://developer.chrome.com/docs/devtools/javascript/breakpoints/)
-2.  Then, watch [this video](https://www.youtube.com/watch?v=JzZFccCEgGA) by The Net Ninja for more detail on using the JavaScript Console.
```

Comment:
It's certainly outdated at this point. That said I would prefer us to link to the documentation for the console and watch the video on it rather than move it entirely. If you could change the line to "- 2.  Then, watch [the console overview video and read through the page] to familiarize yourself with the console and how to use the console(https://developer.chrome.com/docs/devtools/console/)." I would be more than happy to merge the PR.

Reply:
Hi, sure, this would be great!

This link is in the previous section under "console". Would you prefer it moved as a 2nd point? Otherwise "2." would be repetition.

![image](https://user-images.githubusercontent.com/102978943/200289736-50cb35b5-7873-4451-97e1-6960a5c560b0.png)

If the latter it could be something like

```
1.  Head to the [Chrome DevTools Documentation](https://developer.chrome.com/docs/devtools/) by Google. The following subsections cover what you'll be using the Developer Tools for 95% of the time.  Feel free to skip the elements you are already familiar with:
    -  CSS
        1.  [View and change CSS](https://developer.chrome.com/docs/devtools/css/)
        2.  [CSS features reference](https://developer.chrome.com/docs/devtools/css/reference/)
    -  [Get Started With Viewing And Changing The DOM](https://developer.chrome.com/docs/devtools/dom/)
    -  Mobile Simulation
        1.  [Simulate mobile devices with Device Mode](https://developer.chrome.com/docs/devtools/device-mode/)
    -  JavaScript
        1.  [Debug JavaScript](https://developer.chrome.com/docs/devtools/javascript/)
        2.  [Pause your code with breakpoints](https://developer.chrome.com/docs/devtools/javascript/breakpoints/)
        
2. Then, watch [the console overview video and read through the page](https://developer.chrome.com/docs/devtools/console/) to familiarize yourself with the console and its usage.
```

Or we could add the text to the link in "1." and leave just one block of instructions.

Reply:
Sorry about taking a bit to get back to you! Yeah, the latter change would be good. Especially given the importance of understanding the console having them watch the video/read the change is what I was aiming to suggest.

Reply:
It's OK, np!

Then you can review the last file change submitted (`javascript_basics/javascript_developer_tools.md`), which should correspond to the idea.